_config.yml

```diff
@@ -93,6 +93,8 @@ collections:
   cards:
     output: true
     permalink: /card/:name/
+  irl:
```

Comment:
Yeah, with `output: false` I think it will not generate any of these (that's the hope anyway) into the site. We have one page at /irl/spring-2016/ for the whole thing and we loop through the collection docs to generate that page.